### PR TITLE
Rediscovery design version 1.1.0

### PR DESCRIPTION
**Pull request by detig-iii**

### CODE REVIEW

File: htdocs/designs/official/Rediscovery/scripts.js [hunk 1]
```diff
@@ -335,6 +334,10 @@ class App {
     }
   }
 
+  mountSectorMap(){
+
+  }
```
Comment (glaubinix) [inline htdocs/designs/official/Rediscovery/scripts.js:339]: This doesn't do anything?

File: htdocs/content/buildings.php [hunk 2]
```diff
@@ -572,7 +572,7 @@
 
 				while ($tarr = mysql_fetch_array($tres))
 				{
-					tableStart($tarr['type_name'],TABLE_WIDTH);
+					tableStart($tarr['type_name'],TABLE_WIDTH,"","","compact");
```
Comment (glaubinix) [inline htdocs/content/buildings.php:575]: Should this only be for `$cu->properties->itemShow!='full'`? Same as with defense?

Comment (detig-iii) [reply htdocs/content/buildings.php]: Fixed in new commit

File: htdocs/designs/official/Rediscovery/scripts.js (hunk 1 above)
Comment (detig-iii) [reply htdocs/designs/official/Rediscovery/scripts.js]: Fixed in new commit

**Pull request merged**